pkgs/games/phira/default.nix

```
@@ -0,0 +1,57 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, perl
+, wrapGAppsHook
+, openssl
+, alsa-lib
+, at-spi2-atk
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "phira";
+  version = "0.3.3";
+
+  src = fetchFromGitHub {
+    owner = "TeamFlos";
+    repo = pname;
+    rev = "666797936a05743c4e2c45646a7873381210590c";
+    hash = "sha256-JxsZ4UAvYtu5X7qGFLSsi1FgRQ0bpeVDquad4F0Cdsc=";
+  };
+
+  cargoLock = {
+    lockFile = "${src}/Cargo.lock";
```

this wouldn't work since it requires IFD, please vendor `Cargo.lock` instead

```suggestion
    lockFile = ./Cargo.lock;
```